examples/codesandbox/src/App.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { TextProvider } from "text-provider";
+import Banner from "./Banner";
+
+import "./styles.css";
+
+/**
+ * The following object contains the localized
+ * strings against each key. The key is used inside
+ * the React code to display the string,
+ */
+const localizedStrings = {
```

Comment:
Can we load the strings from a JSON file rather than putting them inside the App.js ?

You can use after that:
```js
require('localised-strings.json');
```

Reply:
Extracted strings to a separate JSON file. Have used `import` instead of `require` to be consistent in imports.